common/spec/dependabot/config/ignore_condition_spec.rb

```diff
@@ -121,6 +124,53 @@ def expect_ignored(versions)
         it "ignores expected versions" do
           expect_allowed(minor_upgrades)
           expect_ignored(patch_upgrades + major_upgrades)
+          expect_allowed([dependency_version])
+        end
+      end
+
+      context "with a `major.minor.patch.patch` semver version" do
```

Comment:
```suggestion
      context "with a `major.minor.patch.extra` version" do
```

Nit: `patch.patch` is not a thing, https://semver.org/ . Could we set another name for this non-semver string?